### DISCUSSION TITLE
heap related command support

### DISCUSSION THREAD

**Discussion by derek0883** [pwndbg/pwndbg#962] [Ideas]
seems not support other allocator, no response.

**Comment by gsingh93** [comment]
@derek0883 this may be possible in the near future due to the current refactoring work. Which allocator are you interested in having support for?

**Comment by disconnect3d** [comment]
We are happy to support other allocators but we don't have people that work on such features right now. I am happy to accept/merge support for other allocators if anyone wants to add it. :)

Closing this thread since that's all that could be said here :P.
